kb/2024-07-22/harvester_cluster_shutdown_and_restart.md

```diff
@@ -0,0 +1,576 @@
+
+### (Optional) Backup VMs if Possible
+
+It is always a good practice to backup things before a whole cluster shutdown.
+
+### (Optional) Backup Downstream k8s Clusters if Possible
+
+Harvester doesn't touch the (Rancher manager managed) downstream k8s clusters' workload, when they are not able to be migrated to other node drivers, suggests to backup those clusters.
+
+### (Optional) Stop or Migrate Downstream k8s Clusters if Possible
+
+Harvester doesn't touch the downstream k8s clusters' workload, but suggests to stop or migrate the downstream clusters to avoid your service interruption.
+
+## 3. Shutdown Workloads
+
+### 3.1 Shutdown Traditional VMs
+
+1. Shutdown VM from the VM shell (e.g. Linux `shutdown` command), the OS itself will save data to disks.
+
+2. Check the VM status from [Harvester UI - VM page](https://docs.harvesterhci.io/v1.3/troubleshooting/vm#vm-general-operations), when it is not `Off`, then click the `Stop` command.
+
+### 3.2 Shutdown Rancher Downstream Cluster Machines(VMs)
+
+Suppose your Harvester cluster was [imported to Rancher](https://docs.harvesterhci.io/v1.3/rancher/virtualization-management) as a [node driver](https://docs.harvesterhci.io/v1.3/rancher/rancher-integration#creating-kubernetes-clusters-using-the-harvester-node-driver) before.
+
+When Rancher deploys a downstream cluster on node driver Harvester, it creates a couple of VMs on Harvester automatically. Directly stopping those VMs on Harvester is not a good practice when Rancher is still managing the downstream cluster. For example, Rancher may create new VMs if you stop them from Harvester.
+
+:::note
+
+This depends on the [auto-replace](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke2-cluster-configuration#auto-replace) and/or other options on `Rancher Manager`.
+
+:::
+
+If you have got a solution to **shutdown** those downstream clusters, and check those VMs are `Off`; or there is no downstream clusters, then jump to the step [disable some addons](#33-disable-some-addons).
+
+Unless you have already deleted all the downstream clusters which are deploy on this Harvester, **DO NOT** [remove this imported Harvester from the Rancher manager](https://docs.harvesterhci.io/v1.3/rancher/virtualization-management#delete-imported-harvester-cluster). Harvester will get a different driver-id when it is imported later, but those aforementioned downstream clusters are connected to driver-id.
+
+To safely shutdown those VMs but still keep the Rancher manager managed downstream cluster `alive`, please follow the steps below:
+
+#### Disconnect Harvester from the Rancher Manager
+
+![Rancher and Harvester relationship](./imgs/rancher_manager_harvester.png)
+
+:::note
+
+Harvester has an `embedded rancher` deployment which is used to help the lifecycle management of Harvester itself, it is different from the independently deployed **Rancher manager** for multi-cluster management and more.
```

Comment:
To keep consistency with the wording in the following text below.
```suggestion
Harvester has an embedded `rancher` deployment which is used to help the lifecycle management of Harvester itself, it is different from the independently deployed **Rancher manager** for multi-cluster management and more.
```

Reply:
As this operation also touches the external Rancher server, which is referred as `Rancher manager`; to distinguish from it as explicitily as possbile, the Harvester's internal rancher is called and highlighted as `embedded rancher`.